roadmap/state_sync_protocol/state-sync-protocol.md

```diff
@@ -0,0 +1,204 @@
+# Execution State Synchronization Protocol
+
+## Context
+
+Today, Access nodes act as the gateway to the Flow network, providing a query endpoint for users (REST, gRPC) and performing important admission control duties such as:
+* Rate limiting of requests (script executions, Collection requests, account queries, etc.)
+* Basic validation, proving a first level of filtering to prevent bad transactions from entering the network
+
+Execution nodes and Collection nodes currently still expose public APIs, which are now mostly obsolete. Allowing users to bypass the Access layer and make direct requests to these nodes not only dilutes the valuable resources which should be reserved for performing tasks crucial to the protocol (especially in the case of Execution nodes), but it also leaves them at risk of denial-of-service attacks. Going forward, in order to protect the other staked nodes of the network, the Access layer will become the *only* way for users to interact with the network.
```

Comment:
> Going forward, in order to protect the other staked nodes of the network, the Access layer will become the *only* way for users to interact with the network.

After our latest meeting, are we still opposed to keeping this line?

Reply:
> Execution nodes and Collection nodes currently still expose public APIs, which are now mostly obsolete.

What I mean here is that the *public* APIs are obsolete, meaning there is no need for these APIs to be public since external parties / clients can interact via the Access Node. Suggestions on how to phrase this to make it more clear are welcome.

Reply:
> Going forward, in order to protect the other staked nodes of the network, the Access layer will become the only way for users to interact with the network.

I think that this accurately represents the vision 👍